en/components/snackbar.md

```diff
@@ -183,38 +183,10 @@ public restore() {
 
 <div class="divider--half"></div>
 
-### API Summary
-The following tables summarize some of the useful **igx-snackbar** component inputs, outputs and methods.
-#### Inputs
-The following inputs are available in the **igx-snackbar** component:
+### API References
+In this article we learned how to use and configure the [`IgxSnackbarComponent`]({environment:angularApiUrl}/classes/igxsnackbarcomponent.html). For more details in regards its API, take a look at the links below:
 
-| Name | Type | Description |
-| :--- | :--- | :--- |
-| `actionText` | string | Specifies the text to display for the action button.
-| `autoHide` | boolean | Specifies whether the snackbar should be automatically hidden after `displayTime`. By default, `autoHide` is set to **true**. |
-| `displayTime` | number | Specifies the interval in milliseconds in which the snackbar is visible. By default, `displayTime` is set to 4000 milliseconds. |
-| `isVisible` | boolean | Specifies the component visual state. By default, `isVisible` is set to **false**.|
-| `message` | string | Specifies the displayed message. |
-
-<div class="divider--half"></div>
-
-#### Outputs
-The following events are available in the **igx-snackbar** component:
-
-| Name | Description |
-| :--- | :--- |
-| `animationStarted`  | Emits an event when the snackbar animation starts. |
-| `animationDone`  | Emits an event when the snackbar animation ends. |
-| `onAction`  | Emits an event when an action is executed and provides a reference to the snackbar component as argument. |
-
-#### Methods
-
-The following methods are available for the **igx-snackbar** component:
-
-| Signature       | Description                     |
-| :-------------- | :------------------------------ |
-| `show()`  | Shows the snackbar component and hides it after `displayTime` if `autoHide` is set to **true**. |
-| `hide()`  | Hides the snackbar component. |
+* [`IgxSnackbarComponent`]({environment:angularApiUrl}/classes/igxsnackbarcomponent.html)
```

Comment:
Provide Sassdoc reference for the [igx-snackbar-theme](https://staging.infragistics.local/products/ignite-ui-angular/docs/sass/#function-igx-snackbar-theme)